plenum/test/audit_ledger/helper.py

```diff
@@ -50,8 +50,8 @@ def check_audit_txn(txn,
                 "ppSeqNo": pp_seq_no,
                 "ledgerSize": ledger_size,
                 "stateRoot": state_roots,
-                "primaries": primaries
-
+                "primaries": primaries,
+                "digest": digest,
             },
             "metadata": {
```

Comment:
It would be great to extend unit tests in `plenum/test/audit_ledger/test_audit_ledger_handler.py` with a check that the expected digest is written. As a I understand, currently a default value (empty string) is passed to `do_apply_audit_txn` and checked in `check_audit_txn`. It would be great to pass and check non-default there.

Reply:
I did that in `test_apply_audit_ledger_txn_new_ledger`.

Passed a real non default value to `check_apply_audit_txn` that is checked by `check_audit_txn`.

Do you think that is enough?

Reply:
I am doing this in some places, but I will add a new explicit test for just this.